index.js

```diff
@@ -49,6 +53,9 @@ module.exports = {
   format: function(text, opts) {
     return formatWithShebang(text, normalizeOptions(opts));
   },
+  formatAST: function(ast, opts) {
```

Comment:
Could you duplicate the logic and add it to `__debug` instead. We don't want to make this part of the public API as we're likely going to break it in the future.